pkgs/applications/version-management/gex/default.nix

```diff
@@ -2,25 +2,32 @@
 , stdenv
 , rustPlatform
 , fetchFromGitHub
+, pkg-config
+, openssl
+# libgit2-sys doesn't support libgit2 1.6 yet
```

Comment:
Maybe change this to something like

```
# waiting on gex to update to libgit2-sys >= 0.15
```

(https://github.com/rust-lang/git2-rs/blob/41526ca62894cd68399148e3ec79c3e68c568f1e/libgit2-sys/CHANGELOG.md?plain=1#L31)

Reply:
Sounds fine to me. Changed.